pkg/virtctl/vm/vm.go

```diff
@@ -78,6 +83,8 @@ func NewRestartCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
 			return c.Run(cmd, args)
 		},
 	}
+	cmd.Flags().BoolVar(&forceRestart, "force", forceRestart, "Force Restart VM by deleting the VMI")
+	cmd.Flags().IntVar(&gracePeriod, "grace-period", gracePeriod, "The grace-period time for VMI deletion in seconds (defaults to 0 if not given)")
```

Comment:
This is inconsistent with the param lookup later. unless "grace-period" and "graceperiod" are the same thing?

Reply:
corrected